addons/mail/static/src/models/channel.js

```diff
@@ -0,0 +1,95 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one, many } from '@mail/model/model_field';
+import { insertAndReplace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'Channel',
+    identifyingFields: ['id'],
+    recordMethods: {
+        async fetchChannelMembers() {
+            const channelData = await this.messaging.rpc({
+                model: 'mail.channel',
+                method: 'load_more_members',
+                args: [[this.id]],
+                kwargs: {
+                    known_member_ids: this.channelMembers.map(channelMember => channelMember.partner.id),
```

Comment:
This should now be changed to send the id of channel.partner rather than partner now that it is known.